docs/content/docs/concepts/overview.md

```diff
@@ -30,50 +30,52 @@ Flink Kubernetes Operator acts as a control plane to manage the complete deploym
-|                        | Public Docker repository       | full    | ghcr.io / dockerhub                  |
-|                        | Public Helm repository         | full    | apache release repo                  |
-
+## Core
+- Fully-automated [Job Lifecycle Management](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/custom-resource/job-management/)
+  - Running, suspending and deleting applications
+  - Stateful and stateless application upgrades
+  - Triggering and managing savepoints
+  - Handling errors, rolling-back broken upgrades
+- Multiple Flink version support: v1.13, v1.14, v1.15
+- [Deployment Modes](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/custom-resource/overview/#application-deployments):
+  - Application cluster
+  - Session cluster
+  - Session job
+- Built-in [High Availability](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/ha/kubernetes_ha/)   
+- Extensible validation framework
+  - Webhook and Operator based validation  
+  - [Custom validators](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/validator/)  
+- Advanced [Configuration](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/configuration/) management
+  - Default configurations with dynamic updates
+  - Per job configuration
+  - Environment variables
+- POD augmentation via [Pod Templates](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/custom-resource/pod-template/)
+  - Native Kubernetes POD definitions
+  - Layering (Base/JobManager/TaskManager overrides)
+## Operations
+- Operator metrics: TODO
+- Fully-customizable [Logging](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/metrics-logging/#logging)
+  - Default log configuration  
+  - Per job log configuration
+- Flink Web UI and REST Endpoint Access
+  - Fully supported Flink Native Kubernetes [service expose types](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/#accessing-flinks-web-ui)
+  - Dynamic [Ingress templates](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/ingress/)
+- [Helm based installation](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/helm/)
+  - Automated [RBAC configuration](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/rbac/)
+  - Advanced customization techniques 
+- Up-to-date public repositories 
+  - Helm [https://downloads.apache.org/flink/flink-kubernetes-operator-1.0.0/](https://downloads.apache.org/flink/flink-kubernetes-operator-1.0.0)
+  - Docker [ghcr.io/apache/flink-kubernetes-operator](http://ghcr.io/apache/flink-kubernetes-operator)
 
 # Known issues & limitations
 
-JobResultStore Resource Leak
+## JobManager High-availability
```

Comment:
Do we have some known limitations on JobManager high-availability?

Reply:
Wanted to add that the job restarts when the leader JM goes down.